image.yaml

```diff
@@ -1,12 +1,12 @@
 schema_version: 1
 
-name: "jboss-eap-7/eap-cd-openshift"
+name: "jboss-eap-7/eap-cd-openshift-rhel8"
```

Comment:
@luck3y It's important that the docs folks know about this one!

Reply:
it sort of is and isn't -- we're using the same imagestream names as before, so it should be transparent to the user -- but yeah there may be some mention of the actual image name in the registry that the docs folks may need to know about and I'll mention it to them.